tests/markdown/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -901,11 +901,23 @@ See [CONTRIBUTING.md](CONTRIBUTING.md).
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 # Prettier
 
-[![Gitter](https://badges.gitter.im/gitterHQ/gitter.svg)](https://gitter.im/jlongster/prettier)
-[![Build Status](https://travis-ci.org/prettier/prettier.svg?branch=master)](https://travis-ci.org/prettier/prettier)
-[![Codecov](https://img.shields.io/codecov/c/github/prettier/prettier.svg)](https://codecov.io/gh/prettier/prettier)
-[![NPM version](https://img.shields.io/npm/v/prettier.svg)](https://www.npmjs.com/package/prettier)
-[![styled with prettier](https://img.shields.io/badge/styled_with-prettier-ff69b4.svg)](#badge)
+[![Gitter](https://badges.gitter.im/gitterHQ/gitter.svg)](
+  https://gitter.im/jlongster/prettier
+)
+[![Build Status](https://travis-ci.org/prettier/prettier.svg?branch=master)](
+  https://travis-ci.org/prettier/prettier
+)
+[![Codecov](https://img.shields.io/codecov/c/github/prettier/prettier.svg)](
+  https://codecov.io/gh/prettier/prettier
+)
+[![NPM version](https://img.shields.io/npm/v/prettier.svg)](
+  https://www.npmjs.com/package/prettier
+)
+[![styled with prettier](
+  https://img.shields.io/badge/styled_with-prettier-ff69b4.svg
+)](
+  #badge
```

Comment:
We have some discussion for linebreak in links and decided not to break, see https://github.com/prettier/prettier/issues/3203#issuecomment-342802576.

The proposed change looks better than [my original proposal](https://github.com/prettier/prettier/pull/3204/commits/20421a3164183397bdffaebef2a242e6e7bf9f42#r149656387) but it's out of scope of this PR and still needs more discussion, we should split it out to another issue/PR.

cc @azz, what do you think?

Reply:
Definitely keep these on one line - it looks ugly when it breaks and doesn't work at all on BitBucket server.

Reply:
Should I still break the link reference definitions?

```markdown
[other-ref]:
  https://example.com
  (Shakespeare's "Romeo and Juliet" is a famous play)
```

Reply:
LGTM, but could you open another PR for it? I'd like to keep this PR focus on quotes.

Reply:
👍 ✅ ➡️ #3531